src/NServiceBus.Core/Encryption/Encryptor.cs

```diff
@@ -51,7 +51,7 @@ bool VerifyPrerequisite(FeatureConfigurationContext context)
         static List<PropertyInfo> GetEncryptedProperties(FeatureConfigurationContext context)
         {
             var conventions = context.Settings.Get<Conventions>();
-            return context.Settings.GetAvailableTypes()
+            return context.Settings.GetAvailableMessageTypes()
```

Comment:
My thought was:

Encrypted properties only applies to properties on messages. The code
called the parameters in the lambdas messageType. So my assumption was it
should only apply to messages. In theory the previous code could support
encrypted props on sagas. We even had an issue from a user talking about
that (sorry can't digg I'm on mobile) where we advised against using
encrypted properties on sagas. Happy revert this part if we want to be
defensive

On Tuesday, July 19, 2016, Brandon Ording <notifications@github.com> wrote:

> In src/NServiceBus.Core/Encryption/Encryptor.cs
> <https://github.com/Particular/NServiceBus/pull/3948#discussion_r71401531>
> :
>
> > @@ -51,7 +51,7 @@ bool VerifyPrerequisite(FeatureConfigurationContext context)
> >          static List<PropertyInfo> GetEncryptedProperties(FeatureConfigurationContext context)
> >          {
> >              var conventions = context.Settings.Get<Conventions>();
> > -            return context.Settings.GetAvailableTypes()
> > +            return context.Settings.GetAvailableMessageTypes()
>
> This appears to be the one place in this PR that wasn't previously using
> IsMessageType, so I'm assuming it is actually desirable to scope this to
> message types here as well?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3948/files/aec9bfe3f0bf72dfd5a37ad72986d23783679290#r71401531>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAKossf8VQgduYiAXS_pAKGovIAqluZ7ks5qXR4dgaJpZM4JPpWF>
> .
>